Templating/Helper/NumberHelper.php

```diff
@@ -119,63 +158,115 @@ public function formatCurrency($number, $currency, array $attributes = array(),
+    public function formatScientific($number, array $attributes = array(), array $textAttributes = array(), $symbols = array(), $locale = null)
     {
-        return $this->format($number, \NumberFormatter::SCIENTIFIC, $attributes, $textAttributes, $locale);
+        list($locale, $symbols) = $this->normalizeMethodSignature($symbols, $locale);
+
+        return $this->format($number, \NumberFormatter::SCIENTIFIC, $attributes, $textAttributes, $symbols, $locale);
     }
 
     /**
-     * format a value with its ordinal representation.
+     * Formats a number as ordinal according to the specified locale and
+     * \NumberFormatter attributes.
      *
-     * @param float $number
-     * @param array $attributes
-     * @param array $textAttributes
-     * @param null  $locale
+     * @param string|float|int $number         The number to format.
+     * @param array            $attributes     The attributes used by \NumberFormatter
+     * @param array            $textAttributes The text attributes used by \NumberFormatter.
+     * @param array            $symbols        The symbols used by the formatter.
+     * @param string|null      $locale         The locale used to format the number.
      *
-     * @return string
+     * @return string The formatted number.
      */
-    public function formatOrdinal($number, array $attributes = array(), array $textAttributes = array(), $locale = null)
+    public function formatOrdinal($number, array $attributes = array(), array $textAttributes = array(), $symbols = array(), $locale = null)
     {
-        return $this->format($number, \NumberFormatter::ORDINAL, $attributes, $textAttributes, $locale);
+        list($locale, $symbols) = $this->normalizeMethodSignature($symbols, $locale);
+
+        return $this->format($number, \NumberFormatter::ORDINAL, $attributes, $textAttributes, $symbols, $locale);
     }
 
     /**
-     * format a value with its provided style.
+     * Formats a number according to the specified locale and \NumberFormatter
+     * attributes.
      *
-     * @param float $number
-     * @param int style
-     * @param array $attributes
-     * @param array $textAttributes
-     * @param null  $locale
+     * @param string|float|int $number         The number to format.
+     * @param                  $style
+     * @param array            $attributes     The attributes used by the formatter.
+     * @param array            $textAttributes The text attributes used by the formatter.
+     * @param array            $symbols        The symbols used by the formatter.
+     * @param string|null      $locale         The locale used to format the number.
      *
      * @return string
      */
-    public function format($number, $style, array $attributes = array(), array $textAttributes = array(), $locale = null)
+    public function format($number, $style, array $attributes = array(), array $textAttributes = array(), $symbols = array(), $locale = null)
```

Comment:
Is this a BC break because I used `array` instead of `null` for the default value?